Project 1/styles.css

```
@@ -0,0 +1,20 @@
+.card{
+        max-width: 800px;
+        margin: 30px auto;
+        background-color: #ffffff;}
+
+img{
+    max-width: 100%;}
+    title {
+        font-family: 'Slabo', serif;
+
+}
+body{    
```

Add your `background-color` rule here instead of inline in the HTML:

```suggestion
body {
  background-color: purple;
```